swap_meet/item.py

```diff
@@ -1,2 +1,49 @@
+import uuid
+
+# WAVE 2
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):  # default id is None
+        """
+        Initializes an Item with a unique ID.
+        If no ID is provided, a random unique ID is generated using uuid.
+
+        Written By Mariya Mokrynska
+        """
+        if id == None: 
+            self.id = uuid.uuid4().int  # unique id
+        else:
+            self.id = id
+        self.condition = condition # assign condition value
```

Comment:
When we check for `None` value, we should use `is` instead of `==`. 

If we know that an `id` is not `None`, then we know line 14 will not execute. Therefore, we do not need the `else` block. We should always prefer to reduce the amount of indentation we use and look for ways to cut down on unnecessary conditions.

```suggestion
        if id is None: 
            self.id = uuid.uuid4().int  # unique id
        
        self.id = id
        self.condition = condition # assign condition value
```

Reply:
You could also use a ternary operator to set the id:
```py
self.id = uuid4().int if id is None else id

Reply:
Updated based on the suggested approach. I used a slightly different implementation for line 14: 
id = uuid.uuid4().int

From my understanding, this ensures that self.id = id will always assign either a generated unique ID or the one passed into the constructor.